lib/message_handler.rb

```diff
@@ -15,12 +15,7 @@ class MessageHandler < FeatureFlag
   def initialize(args = {})
     @mark_all_out = args[:mark_all_out] || MarkAllOut.new
     @request_parser = RequestParser.new
-    if args[:response]
-      @response = args[:response]
-    else
-      @response = Response.new
-      @response.setup
-    end
+    @response = args[:response]
```

Comment:
This does cover this case, does it?
```
 9 class EventController < Sinatra::Base
 10   attr_reader :message_handler
 11
 12   def initialize
 13     @message_handler = MessageHandler.new
 14   end
```

Reply:
good catch, it should indeed be: @message_handler = MessageHandler.new(response: Response.new.setup), I added it in a later commit. WIP :)